keep-ui/app/alerts/alerts.tsx

```diff
@@ -16,12 +15,16 @@ import { AlertMethodModal } from "./alert-method-modal";
 import AlertRunWorkflowModal from "./alert-run-workflow-modal";
 
 const defaultPresets: Preset[] = [
-  { name: "Feed", options: [] },
-  { name: "Deleted", options: [] },
-  { name: "Groups", options: [] },
+  { name: "feed", options: [] },
+  { name: "deleted", options: [] },
+  { name: "groups", options: [] },
 ];
```

Comment:
Perhaps we should think about moving this to the "usePreset" hook?

Reply:
I want to make a constants variable in alerts that'll be shared between the components and pages. I was planning on doing it with the `/alerts/` refactor